index.html

```diff
@@ -0,0 +1,177 @@
+          together the community of experts,academics
+          and activists who comprise the Standard code Academy, affiliate network
+          in different countries every year.This year we we hope to expand
+          our invitation list including organisations and individuals.
+        </p>
+
+        <h3>2023.10.18(THU) ~ 2023.10.18(FRI)</h3>
+        <p id="venue">@ Madrid,Spain</p>
+      </section>
+    </div>
+    <section class="program" id="program">
+      <div class="programtitle">
+        <h3>Main Program</h3>
+        <div class="liner"></div>
+      </div>
+      <div class="program-container">
+        <div class="lecture">
+          <div class="lec-image"> <img src="images/speak.png" alt="Program" /></div>
+          <div class="lec-h3">
+            <h3>Lecture</h3>
+          </div>
+          <div class="lec-para">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+          </div>
+        </div>
+        <div class="lecture">
+          <div class="lec-image"><img src="images/exhibit.png" alt="Program" /></div>
+          <div class="lec-h3">
+            <h3>Exhibition</h3>
+          </div>
+          <div class="lec-para">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+          </div>
+        </div>
+        <div class="lecture">
+          <div class="lec-image"> <img src="images/forum.png" alt="Program" /></div>
+          <div class="lec-h3">
+            <h3>Forum</h3>
+          </div>
+          <div class="lec-para">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+          </div>
+        </div>
+        <div class="lecture">
+          <div class="lec-image">
+            <img src="images/wkshop.png" alt="Program" />
+          </div>
+          <div class="lec-h3">
+            <h3>Workshop</h3>
+          </div>
+          <div class="lec-para">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+          </div>
+        </div>
+        <div class="lecture">
+          <div class="lec-image"><img src="images/ignite.png" alt="Program" /></div>
+          <div class="lec-h3">
+            <h3>Algorithms</h3>
+          </div>
+          <div class="lec-para">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+          </div>
+        </div>
+      </div>
+      <button type="button" id="join">Join Conference</button>
+
+      <div class="program-title">
+        <h3>Featured Developers</h3>
+        <div class="liner"></div>
+      </div>
+
+
+
+      <section class="speakers" id="speakers">
+
+      </section>
+      
+
+
+    </section>
+  </main>
+  <footer>
+
+    <div class="patners" id="patners">
+      <h3>Partner</h3>
```

Comment:
- [ ] If you compare the Figma design with yours, you will realize that this partners section is too big, probably due to your images, so please work on this section and reduce the size of this section.